### ISSUE TITLE
check out ma et al 2020 analysis

### ISSUE THREAD

**Issue by lizzieinvancouver** [lizzieinvancouver/decsens#19]
@dbuona I did an analysis of the data deposited with Ma et al 2020 (first events are more sensitive!) but it doesn't seem like logging fixes this issue. Can you take a look at my analyses and make sure I am thinking through it clearly? Check out analyses/ma2020/ma2020quick.R.

**Comment by dbuona** [comment]
You're code looks good to me. I'm trying to think through what issue logging would "fix". I'll keep digging in to it.

**Comment by lizzieinvancouver** [comment]
@dbuona Yeah, I don't think it works here because the GDD has to be greater for later events so the logging would not work. I just needed a mental check.
